README.md

```diff
@@ -13,7 +13,7 @@ pip install -r requirements.txt
 Next, download `chromedriver` from [here](https://chromedriver.storage.googleapis.com/index.html?path=79.0.3945.36/) and move the executable file into the Sourcing directory.
```

Comment:
Update to say mention the task of getting the latest chromedriver executable file

Reply:
Changed in most recent commit.